src/hott/02-homotopies.rzk.md

```diff
@@ -189,8 +189,8 @@ It is sometimes useful to have this in inverse form.
     ind-path
       ( A)
       ( x)
-      ( \ y' p' →
-          triple-concat
+      ( \ y' p' 
+          → triple-concat
```

Comment:
The arrow could be two spaces less indented here

Reply:
I may also note that I didn't intend for the convention to be that the lambda arrow should be placed on the next line as done here. That's not to say I'm against that convention.

Reply:
Right, one limitation with the current implementation is that it does not check whether a given line is indented properly. We will hopefully find an easy way to fix indentations in general.
About the lambda arrow, I seem to have missed that point in the guide and will fix it now.

Reply:
As a temporary fix until proper indentation is implemented, binary operators now move to an indentation level that is one less than the next line originally was.